### ISSUE TITLE
Cypress test "should have an image copyright label with styling" isn't doing what we think

### ISSUE THREAD

**Issue by jimjohnsonrollings** [bbc/simorgh#4379]
**Describe the bug**
I’ve found a Cypress test that seems to not be doing what we think it’s doing

https://github.com/bbc/simorgh/blob/fafc5e6334d815324feadde443466a94daa04bb7/cypress/integration/pages/articles/tests.js#L109-L136

cypress/integration/pages/articles/tests.js:109-136
        it('should have an image copyright label with styling', () => {
          cy.request(`${config[service].pageTypes.articles.path}.json`).then(
            ({ body }) => {
              const copyrightData = getBlockData('image', body);
              const rawImageblock = getBlockByType(
 Show more
<https://github.com/bbc/simorgh|bbc/simorgh>bbc/simorgh | Added by a bot

As far as I can see, this test looks for a paragraph within a figure which contains the text of the copyright field
And if there isn’t a paragraph, it checks that the copyright is BBC
This passes for /news/articles/cn7k01xp8kxo
But fails for /urdu/articles/cwgq7rzv172o
And I think this is because it’s finding the paragraph in the caption inside the figure

And in /news, that caption contains the characters ‘BBC’, but in /urdu does not

If there’s a caption AND a copyright then there’ll be two p objects, which might be why it’s failing
So it sounds like we need to write the test more specifically, rather than it being reliant on the type of HTML it’s looking for
Basically, look for the component, not the HMTL logic

**To Reproduce**
See above

**Expected behavior**
The test doesn't look at the caption.

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**
